packages/code-du-travail-frontend/pages/glossaire/index.js

```diff
@@ -15,6 +15,9 @@ import {
 import { Layout } from "../../src/layout/Layout";
 import Metas from "../../src/common/Metas";
 
+const subtitle =
```

Comment:
camelCase ? :)

Reply:
subtitle is actually an english word so that doesn't bother me this way :)